designer/server/src/main/scala/pl/touk/nussknacker/ui/factory/NussknackerAppFactory.scala

```diff
@@ -37,7 +39,8 @@ class NussknackerAppFactory(processingTypeDataStateFactory: ProcessingTypeDataSt
       _                      <- Resource.eval(IO(SLF4JBridgeHandlerRegistrar.register()))
       metricsRegistry        <- createGeneralPurposeMetricsRegistry()
       db                     <- DbRef.create(config.resolved)
-      feStatisticsRepository <- QuestDbFEStatisticsRepository.create(system, clock)
+      questDbConfig          <- Resource.eval(IO(QuestDbConfig.apply(config.resolved)))
```

Comment:
maybe we should pass the typesafe config to the `createFEStatisticsRepository` method and try to parse it in its body? The quest DB will be better encapsualted